docs/doc/60-contributing/03-rfcs/20220530-user-friendly-releases.md

```diff
@@ -0,0 +1,71 @@
+---
+title: User-friendly Releases
+description: RFC for user-friendly releases
+---
+
+# Summary
+
+A simple user-friendly release solution.
+
+# Motivation
+
+In the past, only specific binaries were included in the Databend release. Understanding how to operate and maintain Databend often requires extensive discussion.
+
+This proposal aims to improve this process by providing some simple tips and default configs to help make the process painless for users.
+
+# Detailed design
+
+The release directory structure is roughly as follows:
+
+```rust
+.
+├── bin
+│   ├── databend-query
+│   ├── databend-meta
+│   └── databend-metactl (?)
+├── configs
+│   ├── databend-query.toml
+│   └── databend-meta.toml
+├── docs 
```

Comment:
Doesn't make sense to me.

- It's hard to read locally.
- It's hard to update and get a quick response (compared to databend.rs)
- It's hard to maintian.

Reply:
> It's hard to read locally.

We have plans to add some comments to make it readable. like clickhouse, but simpler.

> It's hard to update and get a quick response (compared to databend.rs).

Yep, but it seems a little more difficult for the user to write from scratch.

> It's hard to maintian.

This completely reuses [the configs we use for testing](https://github.com/datafuselabs/databend/tree/main/scripts/ci/deploy/config). In fact, I don't think the maintenance problem is caused by this proposal. We have always lacked a more automated solution for document/config generation.

Reply:
Hi, I'm commenting on `docs`.

Reply:
I think it would be very helpful if the release package contains both the toml files with detailed explanations for each parameter in each. So users can change the files accordingly when deploying instead of starting from the ground up. 

I prefer the way Snowflake does for the doc part. Can we add the doc links in the readme file? then we don't need additional tutorials for the package. We update the docs only on our website.

Reply:
> Hi, I'm commenting on `docs`.

Oh, sorry. I'm really not sure what to do with Docs, and markdown is definitely not a good idea. So let's cut it down and just provide some useful links in txt format. How do you feel about it?

PS: github should give me a comment highlight, I did miss it.

Reply:
> Can we add the doc links in the readme file?

I agree. So we can just keep a readme file as doc in release.